core/cli_helpers.py

```diff
@@ -400,9 +400,12 @@ def command_catalog(self):
             'ceph_osd_dump':
                 [BinCmd('ceph osd dump'),
                  FileCmd('sos_commands/ceph/ceph_osd_dump')],
-            'ceph_osd_df_tree':
-                [BinCmd('ceph osd df tree'),
-                 FileCmd('sos_commands/ceph/ceph_osd_df_tree')],
+            'ceph_osd_df_tree_json_decoded':
```

Comment:
While i agree that this provides a much better way to process the information, it seems it is only available in more recent sosreports so maybe we should leave both so that the code can have a fallback?

Reply:
It's available in the current sosreport package of Xenial, Bionic, etc (anyone using sosreport >= 3.8 will have it).

Do you think users might be running w/ older sosreport packages? (we could ask them to upgrade sosreport package, if needed). Otherwise, we're OK.